Platform/DataStructures/Bag.swift

```diff
@@ -167,7 +164,7 @@ extension Bag {
     ///
     /// - parameter action: Enumeration closure.
     func forEach(_ action: (T) -> Void) {
-        if _onlyFastPath {
+        guard !_onlyFastPath else {
```

Comment:
Because it returns after that expression is true, I think a guard statement makes that more clear